openedx/core/djangoapps/discussions/models.py

```diff
@@ -128,6 +130,33 @@ def pii_sharing_required_message(provider_name):
         'messages': [],
         'has_full_support': True
     },
+    'openedx': {
+        'features': [
+            Features.BASIC_CONFIGURATION.value,
+            Features.PRIMARY_DISCUSSION_APP_EXPERIENCE.value,
+            Features.QUESTION_DISCUSSION_SUPPORT.value,
+            Features.COMMUNITY_TA_SUPPORT.value,
+            Features.REPORT_FLAG_CONTENT_TO_MODERATORS.value,
+            Features.AUTOMATIC_LEARNER_ENROLLMENT.value,
+            Features.ANONYMOUS_POSTING.value,
```

Comment:
Why this provider was added? this actually breaks the discussions app on stage/prod. 
https://course-authoring.stage.edx.org/course/course-v1:UBCx+Water201x_2+2T2015/pages-and-resources/discussion/settings

Reply:
@awaisdar001  I think I mentioned this on one of the calls. It's the provider for the new experience. 

I have a PR ready to add support for it in the MFE here: https://github.com/edx/frontend-app-course-authoring/pull/217

I think I should have gotten that merged first, or included this change with a separate PR.

Reply:
@xitij2000 the mentioned PR is not ready yet, can you suggest how we can address this "bad user experience" in the short term? Maybe only revert the provide change in a separate PR?

Reply:
@awaisdar001  I will make a PR that removes this provider right now.

Reply:
https://github.com/edx/edx-platform/pull/29314

Reply:
thanks @xitij2000